content/_code-samples/airgapped-wallet/py/README.md

```diff
@@ -0,0 +1,111 @@
+
+There are channels that can be maliciously used by outside parties to infiltrate an airgapped system and steal sensitive information.
+
+There are other ways malware could interact across airgapped networks, but they all involve an infected USB drive or a similar device introducing malware onto the airgapped machine. (They could also involve a person physically accessing the computer, compromising it and installing malware or modifying its hardware.)
+
+This is why it is also recommended to encrypt sensitive information being stored in an airgapped machine.
+
+The airgapped machine should have a few rules enforced to close any possible channels getting abused to leak information outside of the machine:
+### Wifi
+
+- Disable any wireless networking hardware on the airgapped machine. For example, if you have a desktop PC with a Wifi card, open the PC and remove the Wifi hardware. If you cannot do that, you could go to the system’s BIOS or UEFI firmware and disable the Wifi hardware.
+
+### BlueTooth
+
+- BlueTooth can be maliciously used by neighboring devices to steal data from an airgapped machine. It is recommended to remove or disable the BlueTooth hardware.  
+
+### USB
+
+- The USB port can be used to transfer files in and out of the airgapped machine and this may act as a threat to an airgapped machine if the USB drive is infected with a malware. So after installing & setting up this airgapped wallet, it is highly recommended to block off all USB ports by using a USB blocker and not use them.
+
+Do not reconnect the airgapped machine to a network, even when you need to transfer files! An effective airgapped machine should only serve 1 purpose, which is to store data and never open up a gateway for hackers to abuse and steal data.
+
+# Tutorial
+For testing purposes, you would need to have 2 machines and 1 phone in hand to scan the QR code.
+
+1. 1st machine would be airgapped, following the security practices written [here](#security-practices). It stores and manages an XRPL Wallet. 
+2. 2nd machine would be a normal computer connected to the internet. It relays a signed transaction blob to a rippled node.
+3. The phone would be used to scan a QR code, which contains a signed transaction blob. The phone would transmit it to the 2nd machine.
+
+The diagram below shows you the process of submitting a transaction to the XRPL:
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/87929946/197970678-2a1b7f7e-d91e-424e-915e-5ba7d34689cc.png" width=75% height=75%>
+</p>
+
+# Setup
+- Machine 1 - An airgapped computer (during setup, it must be connected to the internet to download the files)
+- Machine 2 - A normal computer connected to the internet
+- Phone - A normal phone with a working camera to scan a QR
+
+## Machine 1 Setup
+Since this machine will be airgapped, it is best to use Linux as the Operating System.
```

Comment:
I'm not clear on why that is - is it because Linux has better stability without updates? Better in-built security?

Reply:
As compared to Windows, Linux is less prone to malware since a huge number of viruses are developed specifically for Windows, most viruses can't harm Linux in this case.